UPGRADE-2.4.md

```diff
@@ -0,0 +1,53 @@
+# UPGRADE FROM 2.3 to 2.4
+
+## Routing
+
+* The `I18nRouteCollectionBuilder` class has been removed in favor of `I18nRouteGenerator`. 
```

Comment:
On second reading I would suggest either to use FQCN here or use statements in before/after below, wdut?